frontend/src/components/manage-cookies.tsx

```diff
@@ -15,13 +14,7 @@ export const ManageCookiesLink = () => {
         osano.cm.showDrawer('osano-cm-dom-info-dialog-open')
     }
 
-
     return (
-        <>
-            <Global
-                styles={css({ '.osano-cm-widget': { display: 'none' } })}
-            />
-            <a href='#' onClick={showOsano}>Manage Cookies</a>
-        </>
+        <span onClick={showOsano}>Manage Cookies</span>
```

Comment:
if you want to use a span, this needs to have `cursor:hover` and also a role="button" for a18y

Reply:
@nathanstitt the cursor portion is taken care of by the parent link

```
<StyledLink to='#'>
    <Menu.Item>
        <ManageCookiesLink />
    </Menu.Item>
</StyledLink>
```
the result:

![Screenshot 2023-11-13 at 3 23 45 PM](https://github.com/openstax/kinetic/assets/8879108/4037ad53-eb9b-45a4-bb2b-69cee29f5621)

Question is, would we still want the role on the span? or would this parent cover that case?

Reply:
@nathanstitt the cursor portion is taken care of by the parent link

```
<StyledLink to='#'>
    <Menu.Item>
        <ManageCookiesLink />
    </Menu.Item>
</StyledLink>
```
the result:

![Screenshot 2023-11-13 at 3 23 45 PM](https://github.com/openstax/kinetic/assets/8879108/4037ad53-eb9b-45a4-bb2b-69cee29f5621)

Question is, would we still want the role on the span? or would this parent cover that case?

Reply:
nah, sorry I'd left that comment before I realized it was being used in the menu